### PR TITLE
Completed javascript-loops-practice assignment

### PR DESCRIPTION
**Pull request by Aiden-Peace300**
image

### CODE REVIEW

File: javascript-loops-practice/add-suffix-to-all.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+/* exported addSuffixToAll */
+function addSuffixToAll(words, suffix) {
+  for (let i = 0; i < words.length; i++) {
+    words[i] = words[i] + suffix;
+  }
+  return words;
+}
```
Comment (shawnkost) [inline javascript-loops-practice/add-suffix-to-all.js:7]: We don't want to manipulate the `words` array directly. We just want to use this as a reference. I would've created a new array, and pushed the values into that new array. See my solution below.
```
function addSuffixToAll(words, suffix) {
  const result = [];
  for (let i = 0; i < words.length; i++) {
    result.push(words[i] + suffix);
  }
  return result;
}
```

File: javascript-loops-practice/filter-out-nulls.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  let i = 0;
+  while (i < values.length) {
+    if (values[i] === null) {
+      values.splice(i, 1);
+    } else {
+      i++;
+    }
+  }
+  return values;
+}
```
Comment (shawnkost) [inline javascript-loops-practice/filter-out-nulls.js:12]: Again, we don't want to manipulate the `values` array directly, we just want to reference it. We should be utilizing a 2nd array to push values into. Check out my solution
```
function filterOutNulls(values) {
  const result = [];
  for (let i = 0; i < values.length; i++) {
    if (values[i] !== null) {
      result.push(values[i]);
    }
  }
  return result;
}
```

File: javascript-loops-practice/filter-out-strings.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  let i = 0;
+  while (i < values.length) {
+    if (typeof values[i] === 'string') {
+      values.splice(i, 1);
+    } else {
+      i++;
+    }
+  }
+  return values;
+}
```
Comment (shawnkost) [inline javascript-loops-practice/filter-out-strings.js:12]: Same note as above ^ about manipulating the array directly.

**Review by shawnkost — APPROVED** [review]
Approved, but check out the comments I left below!

File: javascript-loops-practice/add-suffix-to-all.js (hunk 1 above)
Comment (Aiden-Peace300) [reply javascript-loops-practice/add-suffix-to-all.js]: Updated. Thank you for letting me know.

File: javascript-loops-practice/filter-out-nulls.js (hunk 2 above)
Comment (Aiden-Peace300) [reply javascript-loops-practice/filter-out-nulls.js]: Updated. Thank you for letting me know.

File: javascript-loops-practice/filter-out-strings.js (hunk 3 above)
Comment (Aiden-Peace300) [reply javascript-loops-practice/filter-out-strings.js]: Updated. Thank you for letting me know.

**Pull request merged**